rrfs-test/testinput_expr/rrfs_mpasjedi_2024052700_Ens3Dvar.yaml

```diff
@@ -49,8 +49,8 @@ cost function:
         active variables: *incvars
         read:
           io:
-            data directory: data/bumploc/conus12km-401km11levels
-            files prefix: bumploc_401km11levels
+            data directory: data/bumploc/conus12km-401km_1p095logp
+            files prefix: bumploc_401km_1p095logp
```

Comment:
This change was mainly a result of my experimenting with BUMP for fv3-jedi. I originally thought that the options for vertical localization in fv3-jedi were sigma and logp (modellevel is definitely not an option). So since this change is primarily made in the interest of keeping a consistent configuration between GSI/FV3-JEDI/MPAS-JEDI during this early stage of testing, I decided to use logp for all test cases. Note that I am not saying that we should use logp for the eventual workflow, just that logp makes sense at this early stage of development. 

But later I found out that logp also does not work for fv3-jedi and only sigma coordinates can be used for vertical localization (see discussion in #53). So then I changed fv3-jedi back to sigma, leaving mpasjedi and GSI with logp coordinates. But since both mpasjedi and GSI can use modellevel as was used in RRFS_A (at least for conventional variables, tracers use logp with VDL), it makes sense to change this back to modellevel. I will make that change. 

Also, RRFS uses 3 modellevels in its retro configuration so this will still need to be changed for MPAS-JEDI no matter what.

Reply:
I changed the vertical localization for MPAS-JEDI back to 11 model levels and GSI back to 3 model levels. The only difference now for MPAS-JEDI is that the bumploc files are generated for 160 mpi tasks. 

For FV3-JEDI, using sigma = 0.04 matches pretty closely with the GSI configuration. However, the MPAS-JEDI vertical localization of 11 model levels does not match the 3 model levels used in GSI. This is because the vertical levels near the surface in MPAS-JEDI are less dense. So the equation used to convert from GSI to JEDI is not valid here. 

However, since the point of unifying these configurations is primarily to test runtime differences, I will leave the configuration as shown in the plots below. Changing the model levels in MPAS-JEDI to match GSI or FV3-JEDI is getting awfully close to tuning territory and I do not think we are interested in that at this stage. We can just be aware that there will be additional differences between MPAS-JEDI and GSI/FV3-JEDI due to the vertical grid spacing and localization. 

![Screenshot 2024-12-20 130854](https://github.com/user-attachments/assets/7f42d3f6-e2ae-431c-bad8-cc0232aef540)

Reply:
@SamuelDegelia-NOAA Thanks for the great tests.

We will update the MPASJEDI case soon to use the recent 65 levels which are more consistent with the RRFSv1 setting. So we don't need to worry about the MPASJEDI part for now.